app/mailers/notification_mailer.rb

```diff
@@ -1,13 +1,15 @@
-# frozen_string_literal: true
-
-# Send notification email based on the email address and message from the db
+require_relative 'application_mailer'
 class NotificationMailer < ApplicationMailer
-  def notification_email(name, email, message)
-    @name = name
-    @email = email
-    @message = message
-    mail(from: 'nitika@wetransfer.com', to: @email, subject:
-        'Click open to view the items expiring soon in your shelves..!!!
-         Time to take action.', message: @message)
+
+  def notification_email
+    @message = ''
+    User.all.each do |user|
```

Comment:
You'll find yourself iterating over a lot of users at some point, so it's safer to use [`find_each`](https://api.rubyonrails.org/classes/ActiveRecord/Batches.html#method-i-find_each) here.